autotest.pm

```diff
@@ -108,6 +108,65 @@ sub _debug_python_version () {
     bmwqemu::diag "Using python version " . $debug;
 }
 
+
+=head2 _openqa_require_lua
+
+    use("testapi");
+    use("some::perl::module", {"some_function", "another_function"});
+
+This function is the importer for loading perl modules from Lua test modules.
+It imports everything that a perl module exports into the global Lua scope.
+Optionally a table listing the desired functions can be passed.
+
+=cut
+
+sub _openqa_require_lua ($perlmod, $import = undef) {
+    my $inf = (defined $import) ? " qw(@$import)" : '';
+    eval("use ${perlmod}${inf}");
+    my @me = (defined $import) ? @{$import} : eval("\@{${perlmod}::EXPORT}");
+    foreach my $fn (@me) {
+        #print "Importing ${perlmod}::${fn}\n";
+        my $pref = '\&';
+        if ($fn =~ m/^\$/) {
+            $pref = '$';
+            $fn =~ s/\$//;
+        }
+        elsif ($fn =~ m/^\%/) {
+            $pref = '\%';
+            $fn =~ s/\%//;
+        }
+        elsif ($fn =~ m/^\@/) {
+            $pref = '\@';
+            $fn =~ s/\@//;
+        }
```

Comment:
It would be nice to have a comment explaining why this code section does what it does.